src/main/java/ch/njol/skript/expressions/ExprSpawnerType.java

```diff
@@ -58,52 +63,43 @@ public EntityData convert(Block block) {
 		}
 		return null;
 	}
-	
-	@Nullable
+
 	@Override
-	public Class<?>[] acceptChange(Changer.ChangeMode mode) {
-		switch (mode) {
-			case SET:
-			case RESET:
+	public Class<?> @Nullable [] acceptChange(ChangeMode mode) {
+		if (mode == ChangeMode.SET || mode == ChangeMode.RESET) {
+			return CollectionUtils.array(EntityData.class);
+		} else if (mode == ChangeMode.DELETE) {
+			if (RUNNING_1_20_0) {
 				return CollectionUtils.array(EntityData.class);
-			default:
-				return null;
+			} else {
+				Skript.error("You can't 'delete' the spawner type of a spawner on Minecraft 1.20.0+");
```

Comment:
```suggestion
				Skript.error("You can only delete the spawner type of a spawner on Minecraft 1.20 or newer");
```
It appears to currently suggest that deletion is only possible on older versions